pkg/query-service/app/logs/v3/query_builder.go

```diff
@@ -330,9 +320,20 @@ func orderBy(items []v3.OrderBy, tags []string) string {
 	for _, item := range items {
 		if item.ColumnName == constants.SigNozOrderByValue {
 			orderBy = append(orderBy, fmt.Sprintf("value %s", item.Order))
+			addedToGroupBy[item.ColumnName] = true
+		}
+	}
+
+	// add the remaining items
+	for _, item := range items {
+		// since these are not present in tags we will have to select them correctly
```

Comment:
You need to handle this based on aggregation type and panel type. Say the aggregation is `SUM` and group by `service_name`. The OrderBy can't be anything other than `value` or `service_name`. But this does seem to add everything.

Reply:
I think jusing keeping this for panel type `list` will sove it.

Reply:
Have added the changes with tests.